bundles/org.eclipse.ui.navigator.resources/META-INF/MANIFEST.MF

```diff
@@ -13,7 +13,7 @@ Export-Package: org.eclipse.ui.internal.navigator.resources;x-internal:=true,
  org.eclipse.ui.internal.navigator.resources.nested;x-internal:=true,
  org.eclipse.ui.internal.navigator.workingsets;x-internal:=true,
  org.eclipse.ui.navigator.resources
-Require-Bundle: org.eclipse.ui.ide;bundle-version="[3.15.0,4.0.0)",
+Require-Bundle: org.eclipse.ui.ide;bundle-version="[3.20.100,4.0.0)",
```

Comment:
You don't need to change that as part of this change. Please remove it.

Reply:
OK, but what about the `org.eclipse.ui.internal.navigator.resources.actions.GotoResourceDialog` which extends the extended `org.eclipse.ui.dialogs.FilteredResourcesSelectionDialog`? No need for "explicit" dependency here?

Reply:
Apart from that, it looks like this will be necessary according to the latest build failure (as tests have been modified as well):

```diff
diff --git a/tests/org.eclipse.ui.tests/META-INF/MANIFEST.MF b/tests/org.eclipse.ui.tests/META-INF/MANIFEST.MF
index 7a3db4f..df3bd11 100644
--- a/tests/org.eclipse.ui.tests/META-INF/MANIFEST.MF
+++ b/tests/org.eclipse.ui.tests/META-INF/MANIFEST.MF
@@ -2,7 +2,7 @@
 Bundle-ManifestVersion: 2
 Bundle-Name: Eclipse UI Tests
 Bundle-SymbolicName: org.eclipse.ui.tests; singleton:=true
-Bundle-Version: 3.15.900.qualifier
+Bundle-Version: 3.15.1000.qualifier
 Eclipse-BundleShape: dir
 Bundle-Activator: org.eclipse.ui.tests.TestPlugin
 Bundle-Vendor: Eclipse.org
```

... and hopefully that could be all? I wonder if these versions checks could be automated on localhost (dev's machine) somehow...

Reply:
resolved, see below